### ISSUE TITLE
[Splitter] Separator elements are missing accessibility attributes

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#3708]
The Splitter split-bar does not have accessible name and is also missing **_aria-valuenow_** attribute:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role#focusable_separator
